src/style.css

```diff
@@ -0,0 +1,156 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+/* #region container */
+.item-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;

```

Comment:
```suggestion
```

flex redundant here, block elements by default located one below other

Reply:
Be more attentive to my comments

![image](https://user-images.githubusercontent.com/63306833/125911501-f45dd141-3d36-4397-aa80-709745da4f05.png)